oeps/architectural-decisions/oep-0005/decisions/0005-centralized-devstack.rst

```diff
@@ -1,26 +1,26 @@
-==========================================
+##########################################
```

Comment:
```suggestion
```
in following what you did with the other ADRs, should this just be a single line of `#`?

Reply:
Note: I'm pretty sure a single-line and double-line of the same character (e.g. `#`) are treated as separate header levels. I'd prefer a single-line (in general), but there are many places where we use double, including OEPs like: oeps/architectural-decisions/oep-0005-arch-containerize-devstack.rst. It probably makes sense to have OEPs and ADRs be consistent.